src/filterlist.js

```diff
@@ -101,6 +101,9 @@ class FilterParams extends React.Component {
     Volume: {
       Default: { ramp_time: 0.0 },
     },
+    Loudness: {
+      Default: { reference_level: 0.0, high_boost: 10, low_boost: 10, ramp_time: 0.0 },
```

Comment:
Could we put 0.2 as default for ramp_time, for both Loudness and Volume? Still fast enough not to be annoying, and avoids abrupt changes.

Reply:
Sounds reasonable